packages/mobile/src/exchange/CeloGoldHistoryChart.tsx

```diff
@@ -158,7 +158,7 @@ function CeloGoldHistoryChart({ t, testID }: Props) {
   ])
   const localExchangeRate = useSelector(getLocalCurrencyExchangeRate)
   const dollarsToLocal = useCallback(
-    (amount) => convertDollarsToLocalAmount(amount, localExchangeRate),
+    (amount) => convertDollarsToLocalAmount(amount, localCurrencyCode ? localExchangeRate : 1),
```

Comment:
Nit: Could we keep things simpler by just having `useSelector(getLocalCurrencyExchangeRate)` return `1` when there is no localCurrencyCode?

Reply:
Yes that would probably work as well. I don't want to make this change in this PR to avoid breaking something else unexpectedly (though unlikely) but as we continue to work on improving local currencies, this will be simplified.